rs/nns/governance/api/src/ic_nns_governance.pb.v1.rs

```diff
@@ -3042,6 +3042,12 @@ pub mod governance {
             Option<governance_cached_metrics::NeuronSubsetMetrics>,
         #[prost(message, optional, tag = "39")]
         pub public_neuron_subset_metrics: Option<governance_cached_metrics::NeuronSubsetMetrics>,
+        #[prost(message, optional, tag = "40")]
+        pub still_losing_voting_power_neuron_subset_metrics:
+            ::core::option::Option<governance_cached_metrics::NeuronSubsetMetrics>,
+        #[prost(message, optional, tag = "41")]
+        pub done_losing_voting_power_neuron_subset_metrics:
```

Comment:
I'm not sure what this means.  Does this mean that it's zeroed out (i.e. not continuing to decline), or that it's recovered by taking action?

Could we make this less ambiguous?  Something like "fully_lost_voting_power" or "recovered_votign_power" or whatever it it's supposed to mean?